lib/vdsm/virt/vm.py

```diff
@@ -5164,14 +5165,9 @@ def onIOError(self, blockDevAlias, err, action):
             self.log.warning('unexpected action %i on device %s error %s',
```

Comment:
Start with uppercase?

Reply:
This is existing log, not added by this commit.